src/Symfony/Bundle/WebProfilerBundle/EventListener/WebDebugToolbarListener.php

```diff
@@ -107,9 +112,73 @@ public function onKernelResponse(FilterResponseEvent $event)
             return;
         }
 
+        if ($this->stopwatch) {
+            $durationByCategory = $this->computeTiming();
+
+            $timing = array();
+            foreach ($durationByCategory as $name => $duration) {
+                $timing[] = sprintf('%s;dur=%.3f;desc="%s"', \preg_replace('/[^a-z0-9#_!#$%&\'*+.^_`|~-]/i', '-', $name), $duration, \addslashes($name));
+            }
+
+            $response->headers->set('Server-Timing', $timing);
```

Comment:
don't we want to aggregate a bit more? ie compact up to 4k instead of generating one header per timing?

Reply:
I replaced `$timing` by `implode(',', $timing)`

Timing is grouped by category wich means ~ 5 entries for most common usage. With an average of 50 byte per entry, I think it's fine to aggregate this header regardless the limitation. Grouping timing will add complexity on that code.